src/python/pants/option/global_options.py

```diff
@@ -898,6 +898,13 @@ def register_bootstrap_options(cls, register):
             help="Whether to use the engine filesystem watcher which registers the workspace"
             " for kernel file change events",
         )
+        register(
+            "--timestamp-format",
```

Comment:
"format" sounds like this option would be an actual format string (which would probably be overkill). Instead, it seems like this is more like "timezone" with options being UTC or auto?

But, I'm not really sure that this needs to be an option. Admins that would like UTC timestamps would generally configure an entire machine as UTC.

Reply:
Agreed on not being an option. Imo, it's generally good how customizable Pants is, but we also run the risk of decision fatigue: https://en.wikipedia.org/wiki/Decision_fatigue.